### PR TITLE
fix(List): update some ui for homepage

### PR DESCRIPTION
**Pull request by youwei91**
This PR is dependence to: https://github.com/seekinternational/srp-react/pull/541

these change is for fulfil the requirement of homepage's job functions

### CODE REVIEW

File: theme/palette/elements.less [hunk 1]
```diff
@@ -34,3 +34,4 @@
 @panelPink: #ff0072;
 @lightBlue: #eef8fc;
 @activeGlow: #94c9e0;
+@listBlue: #206e8f;
```
Comment (mengtzu) [inline theme/palette/elements.less]: There isn't a colour of this name on the Brand Portal, where did it come from?  @patrickliu-sa ?

Comment (youwei91) [reply theme/palette/elements.less]: I also wondering where should I put this color...is from Brian, this blue is for the box-shadow for List item

Comment (patrickliu-sa) [reply theme/palette/elements.less]: @mengtzu That's the darkest blue included in the new blue shade palette. 
blue final

There's another PR pending for that update
https://github.com/seekinternational/seek-asia-style-guide/pull/195

Comment (youwei91) [reply theme/palette/elements.less]: @patrickliu-sa I see...so I guess i better to wait #195 merged I just update this PR

File: react/ListItem/ListItem.less [hunk 2]
```diff
@@ -30,6 +30,7 @@
 }
 
 .hasHoverState:hover {
+  cursor: pointer;
```
Comment (swetharajendra05) [inline react/ListItem/ListItem.less]: just a small clarification, the pointer/cursor is only for hover state ?

Comment (youwei91) [reply react/ListItem/ListItem.less]: I tested, seem like is the same putting under .listItemValue, the outcome is same, what do you think?

**Pull request merged**